### PR TITLE
feat: support gradio client with modelz endpoints

### PR DESCRIPTION
**Pull request by Kaiyang-Chen**

### CODE REVIEW

**Review by gaocegege — COMMENTED** [review]
Could you please add a doc for this?

**Comment by Kaiyang-Chen** [comment]
> Could you please add a doc for this?

Addressed. PTAL.

**Review by gaocegege — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -15,3 +15,37 @@ modelz --help
...
+
+We provide a lightweight Python library that makes it very easy to use any Gradio app served on mmodelz as an API. The functionalities of `GradioClient` are completely identical to `Client` in  `gradio_client` library provided by Gradio. The only difference is that when initialize the client, you should enter your Modelz serving endpoint url instead of a hugging face space.
+
+### Example Usage
+
+```python
+from modelz import GradioClient as Client
+
+# Parameter here is the endpoint of your Modelz deployment
+# The format is like https://${DEPOLOYMENT_KEY}.modelz.tech/
```
[the remark this answers is not in the document]
Comment (Kaiyang-Chen) [reply README.md:31]: ohh, is this because .tech is the suffix for the dev environment？

[the remark this answers is not in the document]
Comment (kemingy) [reply README.md:31]: Yes

[the remark this answers is not in the document]
Comment (Kaiyang-Chen) [reply README.md]: PTAL

**Review by kemingy — APPROVED** [review]

**Pull request merged**